### PR TITLE
Fj 12/setup circle ci

### PR DESCRIPTION
**Pull request by JeremyGibson**
- [x] CI works

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          paths:
+            - "env"
+          key: v7-pip-dependencies-{{ checksum "requirements/test/test.txt" }}-{{ checksum "requirements/base/base.txt" }}
+orbs:
+  slack: circleci/slack@3.4.2
+
+jobs:
+  test:
+    machine:
+      image: ubuntu-1604:202007-01
```
Comment (copelco) [inline .circleci/config.yml:26]: Interesting- why this image?

File: docker-compose.yml [hunk 2]
```diff
@@ -12,7 +12,8 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data
       - ./docker-entrypoint.postgres.sql:/docker-entrypoint-initdb.d/init.sql
-      - ./docker-postgresql.conf:/var/lib/postgresql/data/postgresql.conf
+      - ./docker-postgresql.conf:/etc/postgresql/postgresql.conf
+    command: postgres -c config_file=/etc/postgresql/postgresql.conf
```
Comment (copelco) [inline docker-compose.yml:16]: Hmm, this shouldn't be needed since it works locally in the default location 🤔

File: .circleci/config.yml (hunk 1 above)
Comment (JeremyGibson) [reply .circleci/config.yml:26]: No particular reason. It was one of the available machine images that circle provides. These images come with docker and docker-compose so it is convenient.

File: docker-compose.yml (hunk 2 above)
Comment (JeremyGibson) [reply docker-compose.yml:16]: If that is not in place, the CI database fails on init. Error message says that the `/var/lib/postgresql/data` location is not empty.  Not really sure why it works fine locally.

**Review by copelco — APPROVED** [review]

**Pull request merged**